truss/test_data/test_build_commands/model/model.py

```diff
@@ -0,0 +1,17 @@
+from typing import Any, Dict, List
+
+
+class Model:
+    def __init__(self, **kwargs) -> None:
+        self._data_dir = kwargs["data_dir"]
+        self._config = kwargs["config"]
+        self._secrets = kwargs["secrets"]
+        self._model = None
+
+    def load(self):
+        # Load model here and assign to self._model.
+        pass
+
+    def predict(self, model_input: Any) -> Dict[str, List]:
+        # Invoke model on model_input and calculate predictions here.
+        return {"predictions": [1, 2]}
```

Comment:
optinoal: you can make this prediction check if `example_dir/testing.py` exists to make it a more meaninful test if you want.